src/test/java/com/inhabas/api/service/BoardServiceTest.java

```diff
@@ -0,0 +1,169 @@
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageImpl;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Sort;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.BDDMockito.*;
+
+
+@ExtendWith(MockitoExtension.class)
+public class BoardServiceTest {
+
+    @InjectMocks
+    BoardServiceImpl boardService;
+
+    @Mock
+    NormalBoardRepository boardRepository;
+
+    @Mock
+    MenuRepository menuRepository;
+
+    @Mock
+    MemberRepository memberRepository;
+
+//    @Mock
+//    NormalBoard normalBoard;
+
+//    @Mock
+//    Member member;
+
+    private MockMvc mockMvc;
+
+    @BeforeEach
+    void setUp(){
+        mockMvc = MockMvcBuilders.standaloneSetup(boardService).build();
+    }
+
+    @DisplayName("게시판을 성공적으로 생성한다.")
+    @Test
+    public void createBoard() {
+        //given
+        SaveBoardDto saveBoardDto = new SaveBoardDto("title", "contents", 1, 12201863);
+        NormalBoard normalBoard = new NormalBoard(1, "title", "contents");
+        Menu menu = new Menu(null, 1, null, "name", "description");
+        Member member = new Member(1, "mingyeom", "010-0000-0000","picture", null, null);
+        given(boardRepository.save(any())).willReturn(normalBoard);
+        given(menuRepository.getById(any())).willReturn(menu);
+        given(memberRepository.findById(any())).willReturn(Optional.of(member));
+
+        // when
+        Integer returnedId = boardService.write(saveBoardDto);
+
+        // then
+        then(boardRepository).should(times(1)).save(any());
+        assertThat(returnedId).isNotNull();
+        assertThat(returnedId).isEqualTo(normalBoard.getId());
+    }
+
+    @DisplayName("게시판의 목록을 조회한다.")
+    @Test
+    public void getBoardList() {
+        //given
+        PageRequest pageable = PageRequest.of(0,10, Sort.Direction.ASC, "created");
+
+        BoardDto boardDto1 = new BoardDto(1, "title", "contents", "mingyeom",1, LocalDateTime.now(), LocalDateTime.now() );
+        BoardDto boardDto2 = new BoardDto(2, "title", "contents", "minji",1, LocalDateTime.now(), LocalDateTime.now() );
+
+        List<BoardDto> results = new ArrayList<>();
+        results.add(boardDto1);
+        results.add(boardDto2);
+        Page<BoardDto> boardDto = new PageImpl<> (results, pageable, results.size());
+
+        given(boardRepository.findAllByMenuId(any(), any())).willReturn(boardDto);
+
+        //when
+        Page<BoardDto> boardList = boardService.getBoardList(1, pageable);
+
+        //then
+        then(boardRepository).should(times(1)).findAllByMenuId(any(), any());
+        assertThat(boardList).isEqualTo(boardDto);
```

Comment:
의미를 추론하기 쉽게 작성해주세요~
우리가 나중에 다시 볼때 편하기 위함도 있지만,
우리가 떠나가고 다른 사람이 와서 유지보수 할 때도, 읽기 쉬워야겠죠?
```suggestion
        assertThat(returnedBoardList).isEqualTo(expectedBoardList);
```

좋은 코드는 코드를 읽기만 해도 의미가 와닿는,, 주석이 필요없는 코드가 좋을 코드라고 해요 (by 클린코드)
같이 클린 코드 작성하는 연습 해보도록 합시다~~